cpp/autosar/test/rules/A1-1-3/options.clang

```diff
@@ -0,0 +1 @@
+-Ofast -ffast-math -fgnu-keywords -fno-signed-zeroes -menable-unsafe-fp-math -menable-no-nans -menable-no-infs -menable-unsafe-fp-math -ffinite-math-only -ffloat-store
```

Comment:
Should we extract the flags by qltest into an `options` file? And, should we assume the default `options` file is for clang?

Reply:
As a design decision, I decided it was more consistent to require an explicit options file for a compiler for the purposes of *compiler testing*. I considered just pulling in the options files that exist for clang as default but: 
1) Not all of them are MEANT for clang
2) Doing so would be a historical decision based only on the fact that clang was first. 

Given the low number of options files I decided to requires an explicit `.clang/gcc/etc` file to influence compiler testing.